_data/heroku-landing-page-tabs-images.yml

```diff
@@ -0,0 +1,9 @@
+- id: cloudwatch_logs
+  title: CloudWatch Logs
+  content: /assets/img/landing-pages-content/cloudwatch-logs.png
```

Comment:
These screenshots end up quite blurry and hard to read in the preview env https://deploy-preview-453--keen-clarke-470db9.netlify.app/migrate-from-heroku/#step-3-instrument-your-app... 

Given the tight space we have, I'm trying to think of ideas... 

1. Could we record an animated gif of going to CloudWatch Logs, picking some log stream, and zooming in on the log lines? And similar .gifs for browsing dashboards and alerts? See https://www.datadoghq.com/product/log-management/ (scroll to "Rapid troubleshooting, investigation, and analytics") for an example. 
1. Can the design be tweaked to support bigger screenshots? See https://www.datadoghq.com/product/platform/dashboards/ for an example.
1. Perhaps instead of screenshots, we show a tiny representative image or text, and relevant logos? E.g., For logs, we can show a small snippet (i.e., actual text, not an image) of actual log lines and then a CloudWatch + DataDog logo to the right, to indicate "these are the logging options we support." See the "event viewer" at the top of https://www.papertrail.com/tour/viewer/ for a nice way to show a snippet of log lines.

Again, sorry to be so nitpicky about these, but these images will be a HUGE part of how users to this tiny landing page make a decision, so we need them to work well.

Reply:
yeah - you'e very right here, that this needs to be spot on. I couldn't get it to show properly on my machine earlier, so had to push, so coming back to this now. Some good suggestions there indeed!

Reply:
I'll vote for the animated gif here, as it seems to pull my attention for longer on the datadog website.